website/src/css/custom.css

```
@@ -410,7 +410,8 @@ pre code {
   border: none !important;
   font-size: 0.85rem;
   line-height: 1.6;
-  overflow-x: auto;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)